### PR TITLE
added task announcement component

### PR DESCRIPTION
**Pull request by jameskokoska**
image

### CODE REVIEW

File: client/src/components/task/TaskAnnouncement/TaskAnnouncement.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+.task-check {
+  background-color: var(--green-success);
+  width: 56px;
+  height: 56px;
+  flex-shrink: 0;
+  margin: 0px 0px 0px 20px;
+  border-radius: 15px;
+  outline: 2px solid #ffffff00;
+  transition: outline 500ms, box-shadow 500ms, background-color 500ms, transform 500ms;
```
Comment (natapokie) [inline client/src/components/task/TaskAnnouncement/TaskAnnouncement.scss:43]:
```suggestion
  box-sizing: border-box;
  background-color: var(--green-success);
  width: 56px;
  height: 56px;
  flex-shrink: 0;
  margin: 0px 0px 0px 20px;
  border-radius: 15px;
  border: 2px solid var(--green-success);
  transition: border 500ms, box-shadow 500ms, background-color 500ms, transform 500ms;
```

**Comment by jameskokoska** [comment]
@natapokie donezo!
image

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@jameskokoska can you add a function for `onUnComplete`? There may be a case where someone accidentally checks a task off, and we would want to be able to handle the logic for that separately

**Comment by jameskokoska** [comment]
Going to merge, need check image for checkbox component

**Pull request merged**